PostmonServer.py

```diff
@@ -131,6 +133,7 @@ def verifica_cep(cep):
         except requests.exceptions.RequestException:
             message = '503 Servico Temporariamente Indisponivel'
```

Comment:
O que acha de deixar o cep concatenado na mensagem, mesmo que isso não vá pro airbrake, se for preciso podemos achar o cep olhando os logs

Reply:
@Bcoan mas será que faz sentido para o erro de serviço temporariamente indisponível? O problema neste caso não é o CEP e sim os Correios em sim. Ainda acha necessário concatenar o CEP?

@hugonomura

Reply:
ele só cai aqui caso o correio esteja fora? se for sempre esse caso acho que não precisa mesmo

Reply:
Podemos deixar assim e quando for pensar em expor o CEP talvez valha fazer isto no CepTracker